src/components/ui/DefaultDrawer.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <q-drawer
+    show-if-above
+    bordered
+    side="left"
```

Comment:
use prop for this value

Reply:
There's no need for that. Properties are forwarded automatically.